pilot/cmd/pilot-agent/main.go

```diff
@@ -150,7 +150,7 @@ var (
 	skipParseTokenEnv = env.RegisterBoolVar("SKIP_PARSE_TOKEN", false,
 		"Skip Parse token to inspect information like expiration time in proxy. This may be possible "+
 			"for example in vm we don't use token to rotate cert.").Get()
-	proxyXDSViaAgent = env.RegisterBoolVar("ISTIO_META_PROXY_XDS_VIA_AGENT", false,
+	proxyXDSViaAgent = env.RegisterBoolVar("PROXY_XDS_VIA_AGENT", true,
```

Comment:
Why change the env name? How can it be unmarshaled to NodeMetadata.ProxyXDSViaAgent

Reply:
It isn't, that is why its removed. We don't need to send it to istiod